loadtest/_common.sh

```
@@ -0,0 +1,228 @@
+
+_LT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Python defaults stdout to the ANSI codepage on Windows, which turns the
+# report's non-ASCII punctuation into mojibake mid-table.
+export PYTHONIOENCODING=${PYTHONIOENCODING:-utf-8}
+
+if [ -f "$_LT_DIR/.env" ]; then
+    # Snapshot what the caller already exported and restore it afterwards.
+    # Sourcing alone lets the FILE win, which is backwards: it silently ignores
+    # `PIPESHUB_QUERY= ./perftest.sh ...` and every other one-off override.
+    _lt_preset=$(declare -p $(env | sed -n 's/^\(PIPESHUB_[A-Z_]*\|TOKEN\)=.*/\1/p') 2>/dev/null || true)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)